README.md

```diff
@@ -0,0 +1,77 @@
+# flatcar-packer-qemu
+Packer QEMU image builder for Flatcar Linux
+
+A basic Packer template, Makefile and cloud config file to get started building QEMU images for [Flatcar Linux](https://flatcar-linux.org/).
+
+# Installation
+Verify that you have [Packer](https://www.packer.io/intro/getting-started/install.html) installed.
+
+```
+$ packer -v
+```
+Clone the repository
+```
+$ git clone https://github.com/flatcar-linux/flatcar-packer-qemu.git .
+$ cd flatcar-packer-qemu
+```
+Get Container Linux Config Transpiler
+```
+$ sudo make ct
+```
+
+### Makefile variables
+
+```make
+PACKER_CMD ?= packer
```

Comment:
I think the `Basic example` should come first. Also, a sentence like

> For all options to customize the image properties, see the `Makefile` variables.

is enough IMO.